samples/feedyou/chat-loading.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
```

Comment:
Can you please move this functionality into the `full.html` sample instead? And also you can upload contents of this file to the `feedyou/webchat/chat-loading.html` file in Blob storage?